modules/ixBidAdapter.js

```diff
@@ -1371,6 +1372,16 @@ function buildIXDiag(validBidRequests, fledgeEnabled) {
   return ixdiag;
 }
 
+// Function to check the fledgeAutoConfig settings in pbjs config
+function checkFledgeAutoConfig() {
```

Comment:
Hey @dgirardi We simply want to know when pubs have this config disabled to study adaptability - are you saying paapi.gpt.autoconfig might go away in future or its behaviour change?

Reply:
the default for it will likely change. It also exists only because of an oversight, the current default [requires GPT to load before Prebid](https://github.com/prebid/Prebid.js/issues/10566). I don't understand what use it could have for a bidder, what do you mean by adaptability?

Reply:
@dgirardi This is related to issue: https://github.com/prebid/Prebid.js/issues/11242. Where depending on the configuration, there's a chance of duplicate impressions. Tracking field could help us identify any potential configuration issues, similar to the one mentioned in the issue.  Monitoring this field just provides use with better insights on how fledge is configured.

We can remove this in future releases of PBJS once we agree on what the default value/implementation of `autoconfig` should be should be. Are there any concerns with monitoring this field for now?

Reply:
the concern is that bidders typically do not sniff for and collect configuration that is not related to their bid activity; and we are also discouraging the use of `getConfig` overall, because anything that a bidder needs from it should really be part of the bid request objects, otherwise - when the need is legitimate - other bidders will also need it and end up duplicating code. 

I am having trouble seeing `autoconfig` as legitimate, IMO it's not a good proxy for https://github.com/prebid/Prebid.js/issues/11242, since the scenario describes someone using `setPAAPIConfigForGPT` in an odd and incorrect way, which yes can only be done with a particular `autoconfig` setting but it's hardly the same thing.